package.json

```diff
@@ -75,6 +67,8 @@
     "eslint-plugin-react": "7.30.1",
     "eslint-plugin-react-hooks": "4.6.0",
     "prettier": "2.7.1",
-    "react-error-overlay": "6.0.9"
+    "react-scripts": "4.0.3",
```

Comment:
` "react-scripts": "4.0.3"` should be used in development while `"react-scripts": "5.0.1"` can stay on master

Reply:
Upgraded. 👍